allauthdemo/settings.py

```diff
@@ -0,0 +1,341 @@
+# -*- coding:utf-8 -*-
+import os
+
+import environ
+
+env = environ.Env(
+	DEBUG=(bool, True),
+	CACHE_URL=(str,  'locmemcache://'),
+	EMAIL_URL=('EMAIL_URL', 'smtp://[user]:[password]@[host.domain.tld]:[portnumber]'),
```

Comment:
The unconfigured SMTP settings throw an error when trying to build. This should be mentioned somewhere. Replacing it with the old `EMAIL_URL=(str, 'consolemail://'),` allows the build to complete.

Reply:
I'm not seeing this as a problem if the parameters are completed.